src/HttpRequester.js

```diff
@@ -58,16 +58,19 @@ class HttpRequester {
   }
 
   getContinuationToken(sortByNewest) {
-    let continuation = this.cachedInitialData.match(
+    const result = this.cachedInitialData.match(
       /"itemSectionRenderer".*"token":"([^"]*)".*"targetId":"comments-section"/
-    )[1]
+    )
 
-    // Gets top comments by default, and new comments if true
-    if (sortByNewest) {
-      continuation = continuation.slice(0, 47) + "B" + continuation.substring(48)
+    if (!result) {
```

Comment:
Is this (returning `null`) the fix here?

Reply:
Yes -- before it tried getting result[1] without checking if there was a result. This caused the error. The other changes in this file are a minor refactor.